packages/shared/lib/core/utils/ui.ts

```diff
@@ -159,8 +164,6 @@ export function slidable(node: HTMLElement, use: boolean = true): { destroy: ()
  */
 export function hex2rgb(hex: string): string {
```

Comment:
Looks like the function in `convert.ts` is more specific so I say we keep that one. I created an issue for this #6770 and will create a different PR for that since this one has already grown larger than intended